src/commit.h

```diff
@@ -36,4 +36,11 @@ struct git_commit {
 void git_commit__free(void *commit);
 int git_commit__parse(void *commit, git_odb_object *obj);
 
+typedef enum {
+	GIT_COMMIT_PARSE_QUICK = (1 << 0),
```

Comment:
You should add some documentation here stating what those two flags mean and what information exactly is not going to get parsed for QUICK.

Reply:
Right, will do.